provider/cmd/pulumi-resource-pulumiservice/schema.json

```diff
@@ -564,10 +564,6 @@
     "pulumiservice:index:AccessToken": {
       "description": "Access tokens allow a user to authenticate against the Pulumi Cloud",
       "properties": {
-        "tokenId": {
-          "description": "The token identifier.",
-          "type": "string"
-        },
```

Comment:
Can you explain why we're removing this?

Reply:
> The output tokenId is used as the resourceId, making this one useless

I see this in the description, I'm not sure I understand the implication.

Reply:
I mentioned in description:
- tokenId property is not used anywhere at all, lol. The output tokenId is used as the resourceId, making this one useless

Reply:
Yeah, as I mentioned in the thread, I don't really understand what that's supposed to mean. Why is it useless? What does it mean that it's not really used anywhere?

Reply:
I'll ask the question a different way - what do we gain from making this breaking change?

Reply:
Oh, sorry, didn't update the page and never saw second comment ':D

So the property is literally not used. It's never filled out on create or update, never read from properties or anything. I think it was added originally, then author of AccessToken pivoted to storing tokenId in the resourceId and never cleaned up. That's just a guess as to why though.

We can leave it in to avoid breaking change, no problem with that really, except if a customer goes and tried to access the property and it's empty. Alternatively I can actually fill it out, but that would be storing the same data in 2 places.

So I think options are:
- remove (my preference, breaking change, but not going to impact customers)
- fill (redundant data storing)
- ignore (do nothing, leave it as empty property)

Reply:
Thanks for explaining! Agree with you, makes sense to remove it.